pydicom/tests/test_filereader.py

```diff
@@ -94,6 +94,7 @@
     "emri_small_jpeg_2k_lossless_too_short.dcm"
 )
 color_3d_jpeg_baseline = get_testdata_file("color3d_jpeg_baseline.dcm")
+pm_float32_pixel = get_testdata_file("PM_float32_pixel.dcm")
```

Comment:
The large test file should be added to the [pydicom-data](https://github.com/pydicom/pydicom-data) repo instead.

Also can you add a note to the data/test_files/README file about the source of the dataset and maybe a few details?

Reply:
Alternatively, is a test dataset needed? Can you just mock up a dataset with the elements you need?